src/vi/doctruyen3q/src/eu/kanade/tachiyomi/extension/vi/doctruyen3q/DocTruyen3Q.kt

```diff
@@ -0,0 +1,85 @@
+package eu.kanade.tachiyomi.extension.vi.doctruyen3q
+
+import eu.kanade.tachiyomi.multisrc.wpcomics.WPComics
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.network.interceptor.rateLimit
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.Request
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class DocTruyen3Q : WPComics(
+    "DocTruyen3Q",
+    "https://doctruyen3qk.pro",
+    "vi",
+    dateFormat = SimpleDateFormat("dd-MM-yyyy", Locale.getDefault()),
```

Comment:
Shouldn't be dependent on the environment, use `Locale.ROOT` or some other static locale.